.github/workflows/main.yml

```diff
@@ -34,25 +34,25 @@ jobs:
     - name: Update conda
       run: |
         conda update -y -n base conda setuptools
-    
+
     - name: Init conda
       run: |
         conda init bash
         conda info -a
 
     - name: Create the conda environment
-      run: conda create -q -y -n voila-tests -c conda-forge python=$PYTHON_VERSION pip jupyterlab_pygments==0.1.0 nbconvert=5.5 pytest-cov nodejs flake8 ipywidgets matplotlib xeus-cling
```

Comment:
Don't we still depend on nbconvert for the html rendering?

Reply:
It's in the `setup.py` already.

Reply:
Yes, I realized tha after.